ga4gh/cli.py

```diff
@@ -816,3 +818,59 @@ def client_main(parser=None):
     else:
         runner = args.runner(args)
         runner.run()
+
+##############################################################################
+# Configuration testing
+##############################################################################
+
+
+def run_tests(testObj, tests):
+    """
+    Run a list of tests on a testObj
+    :param testObj: The object under test
+    :param tests: the list of test methods to run
+    :return: a list of results, either None, or a tuple of the test name and
+    reason for failure (raised exception)
+    """
+    def run_test(o, t):
+        try:
+            getattr(o, t)()
+        except AssertionError as e:
+            return t, e
+        else:
+            return None
+    return [run_test(testObj, test) for test in tests]
+
+
+def configtest_main(parser=None):
+    if parser is None:
+        parser = argparse.ArgumentParser(
+            description="GA4GH server configuration validator")
+    parser.add_argument(
+        "--config", "-c", default='DevelopmentConfig', type=str,
+        help="The configuration to use")
+    parser.add_argument(
+        "--config-file", "-f", type=str, default=None,
+        help="The configuration file to use")
+    args = parser.parse_args()
+    configStr = 'ga4gh.serverconfig:{0}'.format(args.config)
+    testConfig = configtest.TestConfig(configStr,
+                                       'GA4GH_CONFIGURATION',
+                                       args.config_file)
+    tests = ['test_config_parameters']
+    try:
+        # setUp can fail, so we wrap it in a try to help it look like
+        # the other tests.
+        testConfig.setUp()
+    except Exception as e:
+        results = [('setUp failed', e)]
+    else:
+        results = run_tests(testConfig, tests)
+
+    logging.basicConfig(level=logging.INFO)
+    log = logging.getLogger(__name__)
+    log.info('{0}/{1} tests passed'.format(results.count(None),
+                                           len(results)))
+    for r in results:
```

Comment:
we tend to frown on single-letter variables here, unless they are counting variables (which this one is not)